InstallSpinnaker.sh

```diff
@@ -3,6 +3,7 @@
 # This script installs Spinnaker and its dependencies.
 # See http://www.spinnaker.io/docs/creating-a-spinnaker-instance
 
+TEMPDIR=$(mktemp -d installspinnaker.XXXX)
```

Comment:
I'd made another comment which I dont think went through.
I'd suggested leaving the rm calls where they were to do partial cleanup in the event of failure. The return at the end wouldnt get called if you only installed dependencies, which terminates early. So the rmdir may as well move up to the end of the dependency installation block, which is where it is used.